11.0/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM debian:stretch-slim
-MAINTAINER Odoo S.A. <info@odoo.com>
+LABEL maintainer  Odoo S.A. <info@odoo.com>
+
+SHELL ["/bin/bash", "-xo", "pipefail", "-c"]
```

Comment:
If you include `-e` here, then the `RUN` lines can use semicolons instead of `&&`, which changes them from a "compound command" to a series of "simple commands" -- Bash parses them slightly differently (especially with regards to looping/control flow structures like `for`, `case`, `while`, etc).

Reply:
Thanks for the clear answers and explanation @tianon